Sources/Sentry/SentryClient.m

```diff
@@ -759,12 +766,31 @@ - (void)applyCultureContextToEvent:(SentryEvent *)event
                   }];
 }
 
-- (void)storeFreeMemoryToDeviceContext:(SentryEvent *)event
+- (void)applyExtraDeviceContextToEvent:(SentryEvent *)event
 {
     [self modifyContext:event
                     key:@"device"
                   block:^(NSMutableDictionary *device) {
                       device[SentryDeviceContextFreeMemoryKey] = @(self.crashWrapper.freeMemory);
+                      device[@"free_storage"] = @(self.crashWrapper.freeStorage);
+
+#if TARGET_OS_IOS
+                      if (self.deviceWrapper.orientation != UIDeviceOrientationUnknown) {
+                          device[@"orientation"]
+                              = UIDeviceOrientationIsPortrait(self.deviceWrapper.orientation)
+                              ? @"portrait"
+                              : @"landscape";
```

Comment:
`l`: We could add a test for this constant here. If I change this line of code, no tests fail.

Reply:
That surprises me, as there is already a test for this in SentryClientTests 🤔
I'll look into this tomorrow.

Reply:
We test for `portrait` I think but not for `landscape`.